README.md

```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
 
 **NOTE**: If Docker of the version less than the required (minimum 20.10.11) is already installed on the server, and/or Docker Compose (minimum 1.29.2), the script will be stopped. You need to update the versions of components manually and run the script again.
 
+#### Features of installation on Azure Virtual machine with Ubuntu
+
+1. Open [Azure Portal](http://portal.azure.com/) and on the search field in the top type in `Virtual machines` and select this item in result search window
+
+1. Click on `+ Create` button in the top left corner and select `Azure Virtual machine` in the menu
+
+1. Select an image of `Ubuntu 18.04` or higher
+
+1. For instance size, select minimum `Standard_F2s_v2 - 2 vCPUs, 4 GiB memory`
+
+1. If you want to access through password, in `Administrator account` section in `Authentication type` menu item select `Password` and setup admin username and password. Otherwise leave `SSH public key` and it will generate the keys after creation
+
+1. In `Inbound port rules` leave `SSH(22)` as by default
+
+1. In the `Networking` tab select existing or create new Virtual Network and Subnet used by this VM
+
+1. Public IP address will be created automatically
+
+1. In `NIC network security group` select `Advanced`
+
+1. In `Configure network security group` click `Create new`
+
+1. Click `+ Add an inbound rule`
+
+1. Add {BakeryPort} (default 3030, where Bareky runs) is the `Destination port ranges`
+
+1. In `Protocol` Choose `TCP`
+
+1. Click `Add`
+
+1. Check that `SSH (TCP/22)` is confugured by default. If not, add it manually using same algorithm as for {BakeryPort}
+
+1. Click `Ok` to finish creating Network security group
+
+1. Click `Review + create`
+
+1. Click `Create`
+
+1. After creating and running the virtual machine, connect to it from outside using admin/password or the SSH protocol, as it was configured.
+
+1. Run this command preferably from `/home` Linux directory to download, install and launch UI Bakery:
+
+   ```bash
+   curl -k -L -o install.sh https://raw.githubusercontent.com/uibakery/self-hosted/main/install.sh && bash ./install.sh
+   ```
+
+1. In the process, upon request, enter the previously received license code, hosting URL and port.
```

Comment:
We don't need to enter port here I bet. Becaseu we've already opened :3030

Reply:
also let's tell where to get hosting URL - it'll be the IP address of the VM

Reply:
We need to enter the port, because it is used now for Bakery installation - same as on local machine

Reply:
Fixed